sdks/java/io/hbase/src/test/java/org/apache/beam/sdk/io/hbase/HBaseIOTest.java

```diff
@@ -98,7 +98,8 @@
     public static void beforeClass() throws Exception {
         conf.setInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER, 1);
         // Try to bind to localhost (this is apparently only for the web console)
```

Comment:
nothing you may want to remove some of the other comments/test hacks around this if these two properties do indeed do the right trick.